fdb-sql-layer-core/src/main/java/com/foundationdb/server/store/PersistitStore.java

```
@@ -396,6 +396,12 @@ protected void trackTableWrite(Session session, Table table) {
     }
 
     @Override
+    Row storeLobs(Session session, Row row) {
+        // does nothing
+        return row;
```

Should probably `throw UnsupportedOperationException()`.